### ISSUE TITLE
Training RetinaNet with MobileNet backbone

### ISSUE THREAD

**Issue by Umengying** [fizyr/keras-retinanet#424]
I'm training RetinaNet with MobileNet224_1.0 backbone on pascal voc 2012+2007trainval, after a few attempts, the performance is still bad(the highest mAP is about 0.45). Here is my trial and error:
1. learning rate: 1e-5, big lr easily goes to NAN.
2. gamma: I have tried `{2, 1, 0.8}`, it seems lower gamma achives higher mAP, but lower gamma easily goes to NAN.
3. according to #340 , I modified `preprocess_image` function in `utils/image.py `
```
def preprocess_image(x):
    x = x.astype(keras.backend.floatx())
    x /= 127.5
    x -= 1
    # convert BGR -> RGB, to keep consistant with MobileNet
    x = x[..., ::-1]
    return x
```
Anyone who can help me? Thanks a lot!

**Comment by liminghuiv** [comment]
I saw mobilenetv2 was successfully used in retinanet:
http://www.timlaibacher.me/post/mobilenetv2retinanet/
